WordPress/src/main/java/org/wordpress/android/ui/sitecreation/domains/compose/DomainItem.kt

```diff
@@ -159,16 +170,18 @@ private fun DomainItemPreview() {
             },
             cost = when {
                 it % 3 == 0 -> Cost.Paid("$${it * 5}")
-                it == 5 -> Cost.OnSale("$${it * 2}", "$${it * 3}")
+                it in 1..2 -> Cost.OnSale("$${it * 2}", "$${it * 3}")
```

Comment:
I was trying to figure out the logic in this function and then realised that it is a construction for `@Preview` reasons 😅

Reply:
Hehe, it does the tricks it needs to do imho, showcasing all possible states in the preview 👍🏻